npm/lib/node-install.ts

```diff
@@ -188,7 +188,7 @@ function maybeOptimizePackage(binPath: string): void {
   // This optimization also doesn't apply when npm's "--ignore-scripts" flag is
   // used since in that case this install script will not be run.
   if (!isYarn()) {
-    const tempPath = path.join(__dirname, 'bin-workerd');
+    const tempPath = path.join(__dirname, "bin-workerd");
```

Comment:
Just curious... why are you replacing the single quotes throughout?

Reply:
To fit with the ESLint rules in the types folder, for consistency